apps/devhub/views.py

```diff
@@ -1640,32 +1582,51 @@ def admin(request, addon):
                   {'addon': addon, 'admin_form': form})
 
 
-def docs(request, doc_name=None, doc_page=None):
-    filename = ''
-
-    all_docs = {'getting-started': [], 'reference': [],
-                'policies': ['submission', 'reviews', 'maintenance',
-                             'recommended', 'agreement', 'contact'],
-                'case-studies': ['cooliris', 'stumbleupon',
-                                 'download-statusbar'],
-                'how-to': ['getting-started', 'extension-development',
-                           'thunderbird-mobile', 'theme-development',
-                           'other-addons'],
-                'themes': ['faq']}
-
-    if doc_name and doc_name in all_docs:
-        filename = '%s.html' % doc_name
-        if doc_page and doc_page in all_docs[doc_name]:
-            filename = '%s-%s.html' % (doc_name, doc_page)
-
-    if not filename:
-        return redirect('devhub.index')
-
-    return render(request, 'devhub/docs/%s' % filename)
-
-
-def builder(request):
-    return render(request, 'devhub/builder.html')
+def docs(request, doc_name=None):
+    mdn_docs = {
+        'getting-started': '',
+        'reference': '',
+        'how-to': '',
+        'how-to/getting-started': '',
+        'how-to/extension-development': '#Extensions',
+        'how-to/thunderbird-mobile': '#Application-specific',
+        'how-to/theme-development': '#Themes',
+        'themes': 'Themes/Background',
+        'themes/faq': 'Themes/Background/FAQ',
+    }
+    if waffle.switch_is_active('mdn-policy-docs'):
+        mdn_docs.update({
+            'policies': 'AMO/Policy',
+            'policies/submission': 'AMO/Policy/Submission',
+            'policies/reviews': 'AMO/Policy/Reviews',
+            'policies/maintenance': 'AMO/Policy/Maintenance',
+            'policies/recommended': 'AMO/Policy/Featured',
+            'policies/contact': 'AMO/Policy/Contact',
+        })
+    if waffle.switch_is_active('mdn-agreement-docs'):
+        # This will most likely depend on MDN being able to protect
+        # pages.
+        mdn_docs.update({
+            'policies/agreement': 'AMO/Policy/Agreement',
+        })
+
+    all_docs = {'policies',
```

Comment:
As you might have guessed from the travis results, this is a syntax error in py26 (the `set literals` is only backported to 2.7 from what I can tell).

As it's not very important to have a `set` here, i'd swith to a simple list or tuple.